docs/wiki/framework/cargo-framework.md

```diff
@@ -118,6 +118,7 @@ Note first arg can be a in-game object or a classname of an object type.
  * Arguments:
  * 0: Item <OBJECT or STRING>
  * 1: Vehicle <OBJECT>
+ * 2: Ignore interaction distance <BOOL>
```

Comment:
Function header has: ` * 2: Ignore interaction distance and stability checks <BOOL>`. Maybe add stability checks part too?

Reply:
I could've sworn that I've copy pasted it directly from source. Maybe I was on older branch 🤔, nice catch!